eng/pipelines/coreclr/crossgen2.yml

```diff
@@ -22,9 +22,10 @@ jobs:
     buildConfig: checked
     platforms:
     - Linux_x64
+    - Linux_arm64
     - OSX_x64
     - Windows_NT_x64
-    - CoreClrTestBuildHost # Either OSX_x64 or Linux_x64
+    - Windows_NT_arm64
```

Comment:
Bad merge?

Reply:
Well, I originally deleted it intentionally as I saw both platforms were already in the list but you're right, I shouldn't be deleting this as at some point it may get switched over to something completely different. Thanks for pointing that out!

Reply:
Restored in 15th commit.